headers/functions/overlay29.h

```diff
@@ -24,18 +29,28 @@ void DungeonRngUnsetSecondary(void);
 void DungeonRngSetSecondary(int i);
 void DungeonRngSetPrimary(void);
 void TrySwitchPlace(struct entity* user, struct entity* target);
+void FloorLoop(int unknown);
+void TrySpawnMonsterAndActivatePlusMinus(void);
+bool FloorIsOver(void);
 void ResetDamageDesc(undefined4* damage_desc);
 bool FloorNumberIsEven(void);
+enum monster_id GetKecleonIdToSpawnByFloor(void);
 void TryActivateSlowStart(void);
 void TryActivateArtificialWeatherAbilities(void);
 bool DefenderAbilityIsActive(struct entity* attacker, struct entity* defender,
                              enum ability_id ability_id, bool attacker_ability_enabled);
 bool IsMonster(struct entity* entity);
 void TryActivateTruant(struct entity* entity);
+bool MonsterCanSpawn(struct monster_id_16 monster_id, bool fail_if_mew);
```

Comment:
Use the plain `enum monster_id` in function signatures; there's no distinction since all registers are 32-bit, and the plain enum looks better in the Ghidra decompiler.